src/estimators/tests/exact_set_test.py

```diff
@@ -93,6 +93,18 @@ def test_less_one_estimator_multiple_for_exact_multi_set(self):
     e = LessOneEstimator()
     self.assertEqual(e([s1, s2])[0], 2)
 
+  def test_less_one_estimator_no_freq1(self):
+    s = ExactMultiSet()
+    s.add_ids([1,1,2,2,3,3,3])
+    e = LessOneEstimator()
+    self.assertEqual(e([s]), [1, 1, 1])
+
+  def test_less_one_estimator_no_freq1or2(self):
+    s = ExactMultiSet()
+    s.add_ids([1,1,1,2,2,2,3,3,3])
+    e = LessOneEstimator()
+    self.assertEqual(e([s]), [0, 1, 2])
```

Comment:
Seems we've already decided to go with e[i]=(cardinality of id's with (f==i+1) but not with (f>=i+1))
but I think this definition would make bloom filter based solution less accurate since previously what appears to be the full cardinality is now defined as (f==1), rather than f>=1
https://github.com/world-federation-of-advertisers/cardinality_estimation_evaluation_framework/blob/freqfw2/src/estimators/bloom_filters.py#L360-L364

Reply:
Thank you again for pointing this out.  I have updated the code to conform to the doc.  Let me know what you think.